packages/sirius-web/backend/sirius-web/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.8"
+services:
+  database:
+    image: postgres:12
```

Comment:
Not directly related to this, but PostgreSQL 12.x will stop being supported in a few months (November 14, 2024): https://www.postgresql.org/support/versioning/